core/src/main/java/org/aerogear/mobile/core/MobileCore.java

```diff
@@ -45,23 +53,87 @@ public Logger defaultLog() {
         };
     }
 
-    @Override
     public void bootstrap(Object... args) {
 
-
         try (InputStream configStream = context.getAssets().open(this.mobileServiceFileName);) {
+
+        Set<String> servicesInitted = new HashSet<>();
```

Comment:
since the method that is called is bootstrap I will rename it to servicesBootstrapped.  It is more directly tied to the code (otherwise people may look for spurious init or start methods.)(I'm avoiding start in Android because onStart has a lot of meanings in the platform)